tests/frontend/test_ops_metaoperation.py

```diff
@@ -98,6 +97,18 @@ def test_create_non_positive_integer(self, prog):
         with pytest.raises(ValueError, match='is not a positive integer'):
             ops.New(1.5)
 
+    def test_create_locked(self, prog):
+        """No new modes can be created in a locked Program."""
+        prog.lock()
+        with pytest.raises(CircuitError, match='The Program is locked, no new subsystems can be created'):
+            ops.New(1)
+
+    def test_delete_locked(self, prog):
```

Comment:
@smite I noticed the addition of this new exception caused a whole bunch of older tests to now fail. Thoughts on how to remedy?

Reply:
Fixed now. Had to explicitly unlock a deepcopy of the original program.